src/components/CalendarMonthGrid.jsx

```diff
@@ -166,6 +187,18 @@ export default class CalendarMonthGrid extends React.Component {
     this.container = ref;
   }
 
+  setMonthHeight(height, i) {
+    if (this.calendarMonthHeights[i]) {
+      if (i === 0) {
+        this.calendarMonthHeights = [height, ...this.calendarMonthHeights.slice(0, -1)];
```

Comment:
it's slightly more efficient here to do `[height].concat(this.calendarMonthHeights.slice(0, -1))`